src/dblib/dblib.c

```diff
@@ -1258,13 +1241,115 @@ tdsdbopen(LOGINREC * login, const char *server, int msdblib)
 		return NULL;
 	}
 
-	connection = tds_read_config_info(dbproc->tds_socket, login->tds_login, g_dblib_ctx.tds_ctx->locale);
-	if (!connection) {
-		dbclose(dbproc);
-		return NULL;
+	// Open connection direct while the server is empty
```

Comment:
I think you can get what you want passing a server as `hostname:port` instead

Reply:
it's a nice way to implement it.